src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/AddTeams.js

```diff
@@ -6,7 +6,8 @@ $(document).ready(function () {
     var divisionCounter = 0;
     var teamCounter = 0;
     var MAX_TEAMS_NUMBER = 0;
-    
+    var groupVisibility = "hidden";
```

Comment:
Why don't you use jQuery to control visibility?

Reply:
ok, we will try to use jQuery